packages/wallet/tailwind.config.js

```diff
@@ -25,8 +26,15 @@ module.exports = {
             dark: '#5932c4'
           }
         }
+      },
+      fontFamily: {
```

Comment:
We may want to make this change in the other apps as well eventually.

@rorysaur Heads up that we're making some typography changes.

Reply:
Cool, I did want to update the typography generally but haven't gotten around to looking into the details, so having this example is great.